### ISSUE TITLE
Route Shape : need to sort the array by sequence

### ISSUE THREAD

**Issue by answerquest** [WRI-Cities/static-GTFS-manager#22]
Bug seen on first time load on heroku (but not on local machine):
![gtfs-manager-bug-shapes](https://user-images.githubusercontent.com/9089552/38763597-b48f6660-3fbc-11e8-9c34-71631e9cdbbe.png)

The shape's sequence order is messed up on loading from DB. Checking db.json, confirmed : the rows are stored as 1,10,100... ie text sorting.
```
  "shapes": {
    "1": {
      "shape_dist_traveled": 0.0,
      "shape_id": "R001_0",
      "shape_pt_lat": 10.110608860405785,
      "shape_pt_lon": 76.34918872659786,
      "shape_pt_sequence": 1
    },
    "10": {
      "shape_dist_traveled": 0.63,
      "shape_id": "R001_0",
      "shape_pt_lat": 10.105114690637768,
      "shape_pt_lon": 76.35012437809694,
      "shape_pt_sequence": 10
    },
    "100": {
      "shape_dist_traveled": 15.880000000000006,
      "shape_id": "R001_0",
      "shape_pt_lat": 9.990707688393124,
      "shape_pt_lon": 76.28752271426649,
      "shape_pt_sequence": 100
    },
    "101": { ...

```
When we upload a fresh shapefile, the ordering is proper. It's also peculiar that just the onward direction shape's order is messed up while the return direction shape is fine. So this seems to be an artefact from the GTFS import mechanism.

In any case, the program needs to have a sorting mechanism when retrieving the data for a shape. This will probably be better done at python end itself.

**Comment by answerquest** [comment]
https://github.com/WRI-Cities/static-GTFS-manager/blob/master/web_wrapper.py#L896
`shapeArray = readTableDB....`
(code at the link may change so do ctrl+f)
Need to sort the shapeArray list.

From from https://stackoverflow.com/a/73050/4355695:
```
sortedShapeArray = sorted(shapeArray, key=lambda k: k['shape_pt_sequence'])
```
this should work.

**Comment by answerquest** [comment]
fixed with v1.2.0
